### PR TITLE
Anthos dashboards support Prometheus formated metrics

### PR DESCRIPTION
**Pull request by leonzz**
For the metrics that will have support for prometheus format, update the graph to display a union for both the old foramt and the new format.

Update the node condition graph to a more meaningful node count.
Also switch the use of KSM metric kube_node_status_capacity to more stable node/cpu/total_cores.

### CODE REVIEW

**Comment by leonzz** [comment]
bring attention to @baizhenyu as well.

**Review by yqlu — APPROVED** [review]

File: dashboards/google-anthos/anthos-cluster-node-status.json [hunk 1]
```diff
@@ -148,7 +131,7 @@
...
+                "timeSeriesQueryLanguage": "{ fetch k8s_node\n| metric 'kubernetes.io/anthos/node_filesystem_free_bytes'\n| add [cluster: resource.cluster_name, node:resource.node_name]\n| group_by 1m,\n    [value_mean:\n       mean(value.node_filesystem_free_bytes)]\n| every 1m\n| group_by\n    [resource.project_id, resource.location, cluster,\n     node, metric.device],\n    [value_mean_aggregate:\n       aggregate(value_mean)]\n;\nfetch prometheus_target\n| metric 'kubernetes.io/anthos/node_filesystem_free_bytes/gauge'\n| add [cluster: resource.cluster, node:resource.instance]\n| group_by 1m,\n    [value_mean:\n       mean(value.gauge)]\n| every 1m\n| group_by\n    [resource.project_id, resource.location, cluster,\n     node, metric.device],\n    [value_mean_aggregate:\n       aggregate(value_mean)]\n}\n| union"
```
Comment (baizhenyu) [inline dashboards/google-anthos/anthos-cluster-node-status.json:134]:
```suggestion
                "timeSeriesQueryLanguage": "{ fetch k8s_node\n  | metric 'kubernetes.io/anthos/node_filesystem_free_bytes'\n  | add [cluster: resource.cluster_name, node: resource.node_name]\n  | group_by 1m, [value_mean: mean(value.node_filesystem_free_bytes)]\n  | every 1m\n  | group_by\n      [resource.project_id, resource.location, cluster, node, metric.device],\n      [value_mean_aggregate: aggregate(value_mean)]\n; fetch prometheus_target\n  | metric 'kubernetes.io/anthos/node_filesystem_free_bytes/gauge'\n  | add [cluster: resource.cluster, node: resource.instance]\n  | group_by 1m, [value_mean: mean(value.gauge)]\n  | every 1m\n  | group_by\n      [resource.project_id, resource.location, cluster, node, metric.device],\n      [value_mean_aggregate: aggregate(value_mean)] }\n| union"
```

File: dashboards/google-anthos/anthos-cluster-on-vmware-vm-status.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
+                "timeSeriesQueryLanguage": "{ fetch k8s_pod::kubernetes.io/anthos/vm_cpu_ready_seconds\n  | align rate(1m)\n  | add\n      [cluster_name:\n         if(re_full_match(resource.namespace_name, 'kube-system'),\n           resource.cluster_name, resource.namespace_name), node:metric.node]\n  | every 1m\n  | group_by\n      [resource.project_id, cluster_name, resource.location, node],\n      [value_vm_cpu_ready_seconds_aggregate:\n         aggregate(value.vm_cpu_ready_seconds)]\n; fetch k8s_node::kubernetes.io/anthos/node/cpu/total_cores\n  | every 1m\n  | add [cluster_name: resource.cluster_name, node:resource.node_name]\n  | group_by\n      [resource.project_id, resource.location, cluster_name, node],\n      [value_total_cores_aggregate:\n         aggregate(value.total_cores)] }\n| join\n| div\n| mul (100)"
```
Comment (baizhenyu) [inline dashboards/google-anthos/anthos-cluster-on-vmware-vm-status.json:12]:
```suggestion
                "timeSeriesQueryLanguage": "{ fetch k8s_pod::kubernetes.io/anthos/vm_cpu_ready_seconds\n  | align rate(1m)\n  | add\n      [cluster_name:\n         if(re_full_match(resource.namespace_name, 'kube-system'),\n           resource.cluster_name, resource.namespace_name), node: metric.node]\n  | every 1m\n  | group_by [resource.project_id, cluster_name, resource.location, node],\n      [value_vm_cpu_ready_seconds_aggregate:\n         aggregate(value.vm_cpu_ready_seconds)]\n; fetch k8s_node::kubernetes.io/anthos/node/cpu/total_cores\n  | every 1m\n  | add [cluster_name: resource.cluster_name, node: resource.node_name]\n  | group_by [resource.project_id, resource.location, cluster_name, node],\n      [value_total_cores_aggregate: aggregate(value.total_cores)] }\n| join\n| div\n| mul (100)"
```

File: dashboards/google-anthos/anthos-cluster-pod-status.json [hunk 3]
```diff
@@ -124,7 +124,7 @@
...
+                "timeSeriesQueryLanguage": "{ fetch k8s_container\n| metric 'kubernetes.io/anthos/kube_pod_status_phase'\n| add [cluster:resource.cluster_name]\n| group_by 1m,\n    [value_phase_mean: mean(value.kube_pod_status_phase)]\n| every 1m\n| group_by\n    [metric.phase, resource.project_id, resource.location,\n     cluster],\n    [value_phase_mean_aggregate:\n       aggregate(value_phase_mean)]\n;\nfetch prometheus_target\n| metric 'kubernetes.io/anthos/kube_pod_status_phase/gauge'\n| add [cluster:resource.cluster]\n| group_by 1m,\n    [value_phase_mean: mean(value.gauge)]\n| every 1m\n| group_by\n    [metric.phase, resource.project_id, resource.location,\n     cluster],\n    [value_phase_mean_aggregate:\n       aggregate(value_phase_mean)]\n}\n| union"
```
Comment (baizhenyu) [inline dashboards/google-anthos/anthos-cluster-pod-status.json:127]:
```suggestion
                "timeSeriesQueryLanguage": "{ fetch k8s_container\n  | metric 'kubernetes.io/anthos/kube_pod_status_phase'\n  | add [cluster: resource.cluster_name]\n  | group_by 1m, [value_phase_mean: mean(value.kube_pod_status_phase)]\n  | every 1m\n  | group_by [metric.phase, resource.project_id, resource.location, cluster],\n      [value_phase_mean_aggregate: aggregate(value_phase_mean)]\n; fetch prometheus_target\n  | metric 'kubernetes.io/anthos/kube_pod_status_phase/gauge'\n  | add [cluster: resource.cluster]\n  | group_by 1m, [value_phase_mean: mean(value.gauge)]\n  | every 1m\n  | group_by [metric.phase, resource.project_id, resource.location, cluster],\n      [value_phase_mean_aggregate: aggregate(value_phase_mean)] }\n| union"
```

**Review by baizhenyu — APPROVED** [review]

**Pull request merged**